cmd/skaffold/app/cmd/parse_config_test.go

```diff
@@ -419,8 +441,11 @@ requires:
 				Profiles:            test.profiles,
 			})
 
-			t.CheckDeepEqual(test.err, err, cmp.Comparer(errorsComparer))
-			t.CheckErrorAndDeepEqual(test.err != nil, err, test.expected, cfgs)
+			if test.err != "" {
+				t.CheckDeepEqual(errors.New(strings.ReplaceAll(test.err, "WORK_DIR", wd)), err, cmp.Comparer(errorsComparer))
+			} else {
+				t.CheckDeepEqual(test.expected, cfgs)
+			}
```

Comment:
Might be easier to have an `duplicateError` type, and instead use a function to check the correctness of the error.

Better yet: create an error-code for this situation and test that error code.  This should be reported upwards.

Reply:
I will make this change in fixing https://github.com/GoogleContainerTools/skaffold/issues/5412

Reply:
The tests are failing as you're getting windows path separators.  Maybe you need to replace `/` &rarr; `\` too in the WORKDIR references.